src/converters/utils/calculations/getApproveTime.ts

```
@@ -42,12 +44,13 @@
 
   const isApproved =
-    statuses.some((status) => status.state === "approved") &&
+    statuses.filter((status) => status.state === "approved").length >=
+      parseInt(getValueAsIs("REQUIRED_APPROVALS")) &&
     !statuses.some((status) => status.state === "changes_requested");
 
   return isApproved
     ? statuses.sort((a, b) =>
         isBefore(parseISO(a.submittedAt), parseISO(b.submittedAt)) ? 1 : -1
-      )[0]?.submittedAt
+      )[parseInt(getValueAsIs("REQUIRED_APPROVALS")) - 1]?.submittedAt
```

Biome (1.9.4)

[error] Use Number.parseInt instead of the equivalent global.

ES2015 moved some globals into the Number namespace for consistency.
Safe fix: Use Number.parseInt instead.


(lint/style/useNumberNamespace)